python/templates/JuliaCollection.jl.jinja2

```diff
@@ -1,2 +1,5 @@
+{% import "macros/params.jinja2" as params %}
 include("{{ class.bare_type }}.jl")
-{{ class.bare_type }}Collection = Vector{ {{ class.bare_type }}{% if params_jl %}{ {{ params_jl | join(',') }} }{% endif %} }
+using .{{ class.bare_type }}Module: {{class.bare_type}}
+
+{{ class.bare_type }}Collection = Vector{ Main.{{ class.bare_type }}Struct{{ params.julia_parameters(params_jl, "Main.", "Struct") }} }
```

Comment:
```suggestion
{{ class.bare_type }}Collection = Vector{ Main.{{ class.bare_type }}Struct{{ params.julia_parameters(params_jl, prefix="Main.", suffix="Struct") }} }
```
Not strictly necessary, but it just makes it very explicit and slightly more readable without having to go to the macro definition.